samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -544,6 +584,39 @@ For example, when building from the command line, you can add it as follows:
 
    west build -b *board_target* -- -DFILE_SUFFIX=release
 
+DFU build with the key storage in KMU
```

Comment:
Can we refer to nRF54L in section name (as it's the only supported SoC Series now)? That might simplify navigation for customers using other SoC family

Reply:
Or maybe we could make `The build targets based on the nRF54L SoC Series are currently the only targets that support the KMU-based key storage.` as a note to highlight it